rpcs3/rpcs3qt/trophy_manager_dialog.cpp

```diff
@@ -708,17 +704,24 @@ void trophy_manager_dialog::PopulateGameTable()
 
 	m_game_combo->clear();
 
+	QList<QString> names;
+	QList<int> indices;
 	for (int i = 0; i < m_trophies_db.size(); ++i)
+	{
+		const QString name = qstr(m_trophies_db[i]->game_name).simplified();
+		m_game_combo->addItem(name, i);
+		names.append(name);
+		indices.append(i);
+	}
+
+	QtConcurrent::blockingMap(indices, [this, names](int& i)
```

Comment:
Since this call will block until all jobs in sequence are done, you probably don't need to copy `names` and accessing it via reference should be enough and potentially faster?